template/tmpl/layout.tmpl

```diff
@@ -137,23 +138,6 @@ $( function () {
 		showMenu : true,
 		enableDoclinks : true
 	} );
-
-	$( "#toc" ).toc( {
-		anchorName  : function ( i, heading, prefix ) {
-			return $( heading ).attr( "id" ) || ( prefix + i );
```

Comment:
this should probably go back in with fix from
 - https://github.com/docstrap/docstrap/pull/89/files
?